src/read.c

```diff
@@ -625,7 +625,7 @@ static void avifDecoderDataDestroy(avifDecoderData * data)
     avifFree(data);
 }
 
-static avifResult avifDecoderReadItem(avifDecoder * decoder, avifDecoderItem * item, avifROData * outData, size_t partialByteCount)
+static avifResult avifDecoderItemRead(avifDecoderItem * item, avifIO * io, avifROData * outData, size_t partialByteCount)
```

Comment:
I renamed the function "avifDecoderItemRead" and made avifDecoderItem * item the first argument. Please take a look. Thanks!

Alternatively, I can rename the function "avifIOReadItem" or "avifIOReadDecoderItem". Would you prefer that?

Reply:
No, this is a nice choice. I'll let it go through the builders, and then merge it.